packages/next/server/render.tsx

```diff
@@ -182,6 +182,19 @@ function enhanceComponents(
   }
 }
 
+function renderApp(
+  App: AppType,
+  Component: React.ComponentType,
+  router: ServerRouter,
+  props: any
+) {
+  if (process.env.__NEXT_RSC) {
```

Comment:
Just curios: why adding a new env variable instead of using the `isServerComponent` parameter passed to `render`?

Reply:
The _app changed for all pages, we have non RSC pages. so when the RSC is enabled we'd like to use the new _app for all.